apis/v1alpha1/httproute_types.go

```diff
@@ -528,7 +528,7 @@ type HTTPRouteForwardTo struct {
 	// Support: Core
 	//
 	// +kubebuilder:default=1
-	// +kubebuilder:validation:Minimum=1
+	// +kubebuilder:validation:Minimum=0
```

Comment:
Do we need a doc line to indicate that the target is ignored if weight is zero or is that self-explanatory?

Reply:
I thought about that briefly but thought it was self explanatory, don't mind adding it in though if that helps clear up anything.

Reply:
I agree with @hbagdi, let's be explicit by having a sentence explaining `weight: 0` ignores the target.

Reply:
What happens when the total weight is 0?

Reply:
Thanks for the feedback! I've updated this to add the following sentence: 

> If weight is set to 0, no traffic should be forwarded for this entry.

I thought about adding one for if the total weight is 0, but every way I tried to say it sounded redundant:

> If all weights in this list are set to 0, no traffic should be forwarded for this forwardTo.

Since this field is optional, implementations already have to support a lack of a `forwardTo` target here, so I don't think dropping the min weight to 0 here significantly changes how this API would be implemented.

Reply:
Yup, I agree the comment here is clear enough